src/c++/library/common.h

```diff
@@ -39,7 +39,9 @@
 #include <string>
 #include <thread>
 #include <vector>
-
+#ifdef TRITON_INFERENCE_SERVER_CAPI_CLASS
+namespace perfanalyzer { namespace clientbackend {class TritonLoader;}}
```

Comment:
The definition is wrapped in an ifdef but is used without the same ifdef elsewhere

Reply:
There should be any reference to perf_analyzer in the C++ library files.

Reply:
You should need to access TritonLoader here as it should be localized to triton_c_api client backend

Reply:
This is in the `c++/library` which does not know about `client_backend` and other related things though?